### PR TITLE
COM-19891: Implement the top menu

### PR DESCRIPTION
**Pull request by SylvainGuittard**
jira issue: https://jira.ez.no/browse/COM-19891

# Improvements
- added bootstrap 3
- added MavenPro
- improved design

# Desktop
ezplatform-top-menu

# iPhone
ezplatform-top-menu-iphone

### CODE REVIEW

**Comment by SylvainGuittard** [comment]
ping @clash82 @damianz5

File: app/Resources/views/pagelayout.html.twig [hunk 1]
```diff
@@ -24,6 +24,17 @@
               href="{{ asset('assets/images/favicon.ico') }}"
               type="image/x-icon" />
 
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+        {% stylesheets filter='cssrewrite'
+        'assets/css/ezplatform-com.css'
```
Comment (clash82) [inline app/Resources/views/pagelayout.html.twig]: you could add indents here :)

File: app/Resources/views/pagelayout_header.html.twig [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-           target="_blank">GitHub</a>
-        |
-        <a href="http://ez-community-on-slack.herokuapp.com/"
-           title="A self-inviter to the eZ Platform Community on Slack: ezcommunity.slack.com"
-           target="_blank">Slack</a>
-        ]
-    </navigation>
+        <div class="collapse navbar-collapse navbar-right" id="navbar">
+            <ul class="nav navbar-nav">
+                <li><a href="doc.ez.no/display/DEVELOPER/Documentation" target="_blank">Docs</a></li>
```
Comment (clash82) [inline app/Resources/views/pagelayout_header.html.twig]: protcol name is missing here `//`

File: app/Resources/views/pagelayout_header.html.twig [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+                <li><a href="//github.com/ezsystems/ezplatform" target="_blank">Github</a></li>
+                <li><a href="ez-community-on-slack.herokuapp.com/" target="_blank">Slack</a></li>
```
Comment (clash82) [inline app/Resources/views/pagelayout_header.html.twig]: same as https://github.com/ezsystems/ezplatform-com/pull/11/files#r76576282
(btw, you can remove ending slash)

File: app/Resources/views/pagelayout_header.html.twig [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
+            </ul>
+        </div>
+    </div>
+</nav>
 
-</header>
+<script>
+    {% autoescape 'js' %}
+    $('#navbar').on('hidden.bs.collapse', function () {
```
Comment (clash82) [inline app/Resources/views/pagelayout_header.html.twig]: indents please :)

File: web/assets/css/ezplatform-com.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    font-family: "Maven Pro";
+    src: url("../fonts/MavenPro-Black.eot"),
+    url("../fonts/MavenPro-Black.ttf") format("truetype"),
+    url("../fonts/MavenPro-Black.svg#MavenProBlack") format("svg"),
+    url("../fonts/MavenPro-Black.woff") format("woff");
+    font-weight: 800;
+    font-style: normal;
+}
+
+
```
Comment (clash82) [inline web/assets/css/ezplatform-com.css]: please remove this line

File: web/assets/css/ezplatform-com.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+body {
+    font-family: 'MavenPro', sans-serif;
```
Comment (clash82) [inline web/assets/css/ezplatform-com.css]: not a big deal but would be nice to use same quotation marks in all styles, I prose to use `"` for CSS styles

File: app/Resources/views/pagelayout_header.html.twig [hunk 7]
```diff
@@ -1,23 +1,36 @@
...
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
 
-    <div class="ez_logo" style="width:90px;height:40px;"><!-- style tag is temporary until CSS is complete -->
-        <img src="{{ asset('assets/images/ez-logo.svg') }}" />
-    </div>
+            <a class="navbar-brand" href="//ezplatform.com/">
```
Comment (clash82) [inline app/Resources/views/pagelayout_header.html.twig]: ending slash is redundant

File: app/Resources/views/pagelayout_header.html.twig (hunk 2 above)
Comment (SylvainGuittard) [inline app/Resources/views/pagelayout_header.html.twig]: About ```Docs```, this is a temporary implementation. In the future, the translation will come from the software.

Comment (clash82) [inline app/Resources/views/pagelayout_header.html.twig]: Even thought it's a temporary solution it should be translated IMO :)

**Comment by damianz5** [comment]
+1

**Comment by clash82** [comment]
+1

**Pull request merged**